packages/core/admin/admin/src/content-manager/pages/App/index.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { Switch, Route, useRouteMatch, Redirect, useLocation } from 'react-router-dom';
 import { CheckPagePermissions, LoadingIndicatorPage, NotFound } from '@strapi/helper-plugin';
 import { DndProvider } from 'react-dnd';
-import HTML5Backend from 'react-dnd-html5-backend';
+import { HTML5Backend } from 'react-dnd-html5-backend';
```

Comment:
Can you put the DnDProvider in the core/admin/pages/Admin/index.js so it available for all plugins?